### PR TITLE
Ignore disk validation if there are no disks

### PR DESCRIPTION
**Pull request by akrejcir**
The validation rules for disks should only be executed
if thre are some disks.

### CODE REVIEW

File: templates/win2k12r2-deprecated.tpl.yaml [hunk 1]
```diff
@@ -43,13 +43,15 @@ metadata:
         }, {
           "name": "windows-virtio-bus",
           "path": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
+          "valid": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
```
Comment (MarSik) [inline templates/win2k12r2-deprecated.tpl.yaml:46]:
```suggestion
          "valid": "jsonpath::.spec.domain.devices.disks[*].disk",
```

The validation should be applied even when the user forgets to set the bus.

Comment (akrejcir) [reply templates/win2k12r2-deprecated.tpl.yaml]: Changed.

**Comment by akrejcir** [comment]
The duplicated travis check looks like a github bug. Both checks point to the same build, which passed.

**Pull request merged**